src/core/connect-extension/connect-extension.ts

```diff
@@ -1,6 +1,6 @@
 import { HttpClient } from '../utils/http-client';
 import CONFIG from '../../config';
-import { encrypt } from '../secure/encrypt.web';
+import { encrypt } from '../secure/encrypt/encrypt.web';
```

Comment:
.web import on non .web.ts file? You missed it?

Reply:
For now it's ok.
It's going to be refactored in another PR.